xbmcaddon.py

```diff
@@ -225,9 +252,48 @@ def __load_add_on_xml(self):
 
         with io.open(add_on_xml, encoding='utf-8') as fp:
             xml_content = fp.read()
-            self.__version = re.findall(r'version="(\d+.\d+.\d+[^"]*)', xml_content)[0]
+            self.__version = re.findall(r'<addon.*?version="(\d+(?:\.\d)?(?:\.\d)?[^"]*)', xml_content)[0]
```

Comment:
I guess this one now also needs a `re.DOTALL` as it might need to match a line break

Reply:
yes, i aggree, I hadn't thought about it.